### PR TITLE
Added config files for testing code in Docker

### PR DESCRIPTION
**Pull request by SanichKotikov**
Please, also check changes in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ JsBarcode has tests on all barcode symbologies (and more). If you want to test y
 
 There are also visual tests located under ``test/browser/``.
 
+Using Docker
+----
+If you have installed [Docker](https://www.docker.com), you can avoid local installation of dependencies for [node-canvas](https://github.com/Automattic/node-canvas). For doing that, run ``npm install --ignore-scripts`` instead of ``npm install``. Then you can run tests by ``docker-compose up``
```
Comment (lindell) [inline CONTRIBUTING.md]: Running any type of npm command outside of docker is not necessary?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM node:8
+
+RUN apt-get update && apt-get install -qq libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev build-essential g++
+
+RUN mkdir -p /frontend
```
Comment (lindell) [inline Dockerfile]: `frontend` does not seem like the correct name for this.

Comment (lindell) [reply Dockerfile]: Sounds good 😃 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (SanichKotikov) [reply CONTRIBUTING.md]: `npm install` will fail if you don't have installed node-canvas' dependencies.

**Pull request merged**